dts/bindings/ethernet/xlnx,axi-ethernet-base.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2024 CISPA Helmholtz Center for Information Security
+# SPDX-License-Identifier: Apache-2.0
+
+description: Xilinx AXI 1G/2.5G Ethernet Subsystem base bindings.
+
+include: [base.yaml]
```

Comment:
don't you want to include ethernet-controller.yaml instead? It also has base.yaml in it.

Reply:
I did not realize this existed, I have changed it accordingly.